content/docs/command-reference/plots/show.md

```diff
@@ -174,10 +174,60 @@ file:///Users/usr/src/plots/logs.csv.json
 
 ```json
 {
-    "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
-    "data": {
-        "values": [
-    {
-        "accuracy": "0.9418667",
-    ...
+  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
+  "data": {
+    "values": [{ "accuracy": "0.9418667", ... }]
+  }
+}
```

Comment:
No need to change this.

Reply:
OK, reverted. (Was only to fix the sea of red in editor from the open bracket, but now the examples that followed are reverted too.)

Reply:
Sounds like an editor issue.